index.js

```diff
@@ -118,7 +118,7 @@ LiveServer.start = function(options) {
 		}
 	});
 	// Output
-	var browserURL = "http://127.0.0.1:" + port;
+	var browserURL = "http://127.0.0.1:" + port + (options.open || '');
```

Comment:
Since the new "open path" only affects the URL opened by the browser, I think it should only be appended to the `open()` command. That's because live-server is still serving the root directory at the root URL, so the console print would be somewhat wrong in stating "Serving (root) at (some subdirectory of root)".